styles.css

```diff
@@ -87,12 +87,13 @@ body {
 .date {
   font-size: 1.8rem;
   font-weight: 800;
-  color: #df7b2e;
+  color: #272A31;
```

Comment:
According with the behance design. I changed the font color with its hexadecimal code for date section